init.sql

```diff
@@ -1050,6 +1077,18 @@ WHERE
     )
 ON CONFLICT DO NOTHING;
 
+
+
+INSERT INTO search_preference_ethnicity (person_id, ethnicity_id)
+    SELECT
+        person.id,
+        ethnicity.id
+    FROM
+        person
+    CROSS JOIN
+        ethnicity
+ON CONFLICT (person_id, ethnicity_id) DO NOTHING;
```

Comment:
I'll make these changes when I merge the PR:

* Move this to the bottom of the file
* Only run this if `search_preference_ethnicity` is empty